src/RONRegistrarController.sol

```diff
@@ -228,6 +228,34 @@ contract RONRegistrarController is
     _transferRONToTreasury();
   }
 
+  /**
+   * @inheritdoc IRONRegistrarController
+   */
+  function bulkRenew(string[] calldata names, uint64[] calldata durations) external payable whenNotPaused nonReentrant {
```

Comment:
User can renew the rns of others. Is that expected?

Reply:
Ýe

Reply:
🚀